infrasim/qemu.py

```diff
@@ -10,19 +11,29 @@ def get_qemu():
         raise Exception("Qemu install Error")
     return qemu_cmd.strip(os.linesep)
 
+def create_macvtap(idx, nic, mac):
+    run_command("ip link add link {} name macvtap{} type macvtap mode bridge".format(nic, idx))
```

Comment:
What if this command failed?  it is suggested to check the run_command return value, if failed, then don't need to execute the next step.